packages/drafts-realm/dropdown.gts

```diff
@@ -0,0 +1,57 @@
+import {
+  FieldDef,
+  field,
+  contains,
+  StringField,
+  containsMany,
+} from 'https://cardstack.com/base/card-api';
+import { Component } from 'https://cardstack.com/base/card-api';
+import { BoxelDropdown, Menu } from '@cardstack/boxel-ui/components';
+import { menuItemFunc } from '@cardstack/boxel-ui/helpers';
+
+class Edit extends Component<typeof Dropdown> {
+  <template>
+    <BoxelDropdown>
+      <:trigger as |bindings|>
+        <button {{bindings}}>
+          {{if @model.selectedValue @model.selectedValue 'Please select'}}
+        </button>
+      </:trigger>
+      <:content>
+        <Menu @items={{this.menuItems}} />
+      </:content>
+    </BoxelDropdown>
+  </template>
+
+  get menuItems() {
+    return this.args.model.displayValues?.map((v: string) =>
+      menuItemFunc([v, () => (this.args.model.selectedValue = v)], {
+        selected: this.args.model.selectedValue === v,
+      }),
+    );
+  }
+}
+
+export class Dropdown extends FieldDef {
```

Comment:
Perhaps its worth discussing this. I often find myself thinking of fields as types; bcos they usually come with some Edit validation. Would it make sense for this to be enum and be a base card?

Reply:
I'm fine with this. its stil in drafts. When we see this re-used many places then let's move it to base. Defer the decision to create a base card for now.

Reply:
I ended up making this class local, but if people decide they want to use it they can export it or move it out, or we can make a fully-fledged base package version then